src/utils/fsWrappers.ts

```diff
@@ -44,6 +44,14 @@ export function tmpdir(): string {
   return os.tmpdir();
 }
 
+/** Wrapper for fs.readFileSync() */
+export function readFileSync(
+  path: fs.PathOrFileDescriptor,
+  options?: { encoding?: BufferEncoding; flag?: string } | BufferEncoding,
+): string {
+  return fs.readFileSync(path, options || "utf-8") as string;
+}
```

Comment:
Not using `readFile` from further up since `~/.docker.config.json` likely won't be in the existing VS Code workspace.